config/bonfire_common.exs

```diff
@@ -43,3 +53,9 @@ config :ex_cldr,
   default_locale: default_locale,
   default_backend: Bonfire.Common.Localise.Cldr,
   json_library: Jason
+
+# Choose password hashing backend
+# Note that this corresponds with our dependencies in mix.exs
+hasher = if config_env() in [:dev, :test], do: Pbkdf2, else: Argon2
```

Comment:
In extensions, these config files with the extension name are intended as documentation for what an app using this extension may need to include in its config (since mix apps ignore the config of their dependencies). So I'd put anything else in `config.exs`

Reply:
I should note this relevant open issue: https://github.com/bonfire-networks/bonfire-app/issues/849
I am planning to use https://hexdocs.pm/igniter/ which provides `install` and `upgrade` commands for setting up dependencies, which could copy over migrations and configs etc that are needed without having to copy paste them manually everywhere for running tests in CI.

Reply:
Thank you for pointing that out, I should have addressed that.